### PR TITLE
Change arguments in the big fix

### PR DESCRIPTION
**Pull request by stephenmwilkins**
A few changes rolled into one since they emerged while I was working on the main one:
- Replaced `synthesizer_data_dir` argument with `grid_dir` and `input_dir` to avoid use of implicit subdirectories.
- bc03-2016 now completely works. To work the this needs to convert the binary files into ascii. This currently works by downloading some fortran code, compiling it, and running it. Running all the bc03-2016 grids is slow, due to the photon luminosity step.
- bpass scripts now work except for the download functionality.
- Made some further modifications to maraston scripts to harmonise with `synthesizer` (e.g. consistent file names).
- Docs modified to reflect this.

## Issue Type

- [x] Bug
- [x] Document
- [x] Enhancement

## Checklist
- [x] I have read the [CONTRIBUTING.md]()
- [x] I have added docstrings to all methods
- [x] I have added sufficient comments to all lines
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no pep8 errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by WillJRoper** [comment]
Of course for now the BPASS scripts won't run regardless.

**Comment by christopherlovell** [comment]
Could you double check that these files have been run through black? I can see a couple instances where there are non compliant formatting issues. We will be moving to this soon so best to do now

**Review by christopherlovell — CHANGES_REQUESTED** [review]
Could you double check that these files have been run through black? I can see a couple instances where there are non compliant formatting issues. We will be moving to this soon so best to do now

**Comment by stephenmwilkins** [comment]
> Could you double check that these files have been run through black? I can see a couple instances where there are non compliant formatting issues. We will be moving to this soon so best to do now

I believe this is now fixed (swapped to using black instead of autopep8 in VSCode). I have not completely fixed the install_bpass<version> since they still don't work.

**Comment by christopherlovell** [comment]
> > Could you double check that these files have been run through black? I can see a couple instances where there are non compliant formatting issues. We will be moving to this soon so best to do now
> 
> I believe this is now fixed (swapped to using black instead of autopep8 in VSCode). I have not completely fixed the install_bpass since they still don't work.

Is this still a Hoki issue? I believe @WillJRoper fixed this, unless it hasn't been merged yet

**Comment by stephenmwilkins** [comment]
@christopherlovell @WillJRoper has a PR that hasn't been accepted yet with no response from the authors. We might have to re-write our own parser.

**Comment by stephenmwilkins** [comment]
@WillJRoper noted that we could force the version of astropy while hoping Hoki update.

File: src/synthesizer_grids/cloudy/create_single_cloudy_input.py
[the remark this answers is not in the document]
Comment (christopherlovell) [reply src/synthesizer_grids/cloudy/create_single_cloudy_input.py:1]: I quite like this. Good for testing as you say

**Pull request merged**